src/components/page-header/page-header.component.less

```diff
@@ -112,21 +112,6 @@ ux-page-header {
             }
         }
 
-        .page-header-navigation {
```

Comment:
Since the secondary masthead is only used in the Micro Focus theme, no styling changes are required here.